client/webpack.config.js

```diff
@@ -2,7 +2,7 @@
  **** WARNING: No ES6 modules here. Not transpiled! ****
  */
 
-/* eslint-disable import/no-extraneous-dependencies */
+/* eslint import/no-extraneous-dependencies: [ "error", { packageDir: __dirname/.. } ] */
```

Comment:
@sirreal `packageDir: __dirname/..`, without any quotes, is not a valid JS or JSON. How can it work? 🙂

Reply:
Magic! Honestly, I don't know, I just played in my editor until it worked which, surprisingly, was this thing that doesn't look valid. Didn't work:

```js
/* eslint import/no-extraneous-dependencies: [ "error", { packageDir: __dirname/ + ".." } ] */
/* eslint import/no-extraneous-dependencies: [ "error", { packageDir: path.dirname( __dirname ) } ] */

// Notably this also didn't work when we point the same style two levels up, so we know that eslint _is_ working at parsing this:
/* eslint import/no-extraneous-dependencies: [ "error", { packageDir: __dirname/../.. } ] */
```

🤷‍♂ 🙈

Reply:
Apparently ESLint has a permissive parser that understands a syntax that's not JS or JSON.